package.json

```diff
@@ -34,5 +39,12 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cypress": "^4.1.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.5",
+    "react-test-renderer": "^17.0.1",
```

Comment:
Why are you using this?

Reply:
We don't know where it came from...we didn't add it...😂
But we should definitely have checked it before we submitted.
Now it's deleted!